schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+  id INT GENERATED BY DEFAULT AS IDENTITY,
+  name VARCHAR NOT NULL,
+  date_of_birth DATE NOT NULL,
+  escape_attempts INT NOT NULL,
+  neutered bit NOT NULL,
```

Comment:
- [ ] Nice job creating the animals table however the type of `neutered` should be **boolean**, kindly make sure to fix this column

Reply:
Done!